twilio_sms/twilio.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+if (process.argv.length <= 3) {
+  console.log("Usage: " + __filename + " <twilio_account_sid> <twilio_auth_token> <from_number> <to_number>");
+  process.exit(-1);
+}
+
+const express = require('express');
+const body_parser = require('body-parser');
+const app = express();
+const PORT = 6691
+
+const [accountSid, authToken, from, to] = process.argv.slice(2)
+const client = require('twilio')(accountSid, authToken);
+
+app.use(body_parser.json())
+app.all('*', function (req, res) {
+  let log = req.body
+  let amount = parseInt(log.topics[1], 16);
```

Comment:
@se3000 It would be ideal if we didn't have to parse out the amount sent ourselves. Maybe it can be done with an existing adapter? Lmk.

Reply:
Yeah, there's nothing yet. I think shortest path would be in solidity to add support for`add(string,uint256)` in the `Chainlinked` contract, and then it would be really easy to pass up and parse out.